standard/namespaces.md

```diff
@@ -446,10 +449,130 @@ Because names may be ambiguous when more than one imported namespace introduces
 > ```
 > *end example*
 
+Furthermore, when more than one namespace or type imported by *using_namespace_directive*s or *using_static_directive*s in the same compilation unit or namespace body contain types or members by the same name, references to that name as a *simple_name* are considered ambiguous.
+
+> *Example*:
+> ```csharp
+> namespace N1
+> {
+>     class A {}
+> }
+> class C
```

Comment:
A blank line between each top-level declaration? This is a bit hard to read as-is. (We might want to talk about this - it looks like a lot of examples don't have lines between top-level declarations.)

Reply:
I did add them here.
I didn't resolve the conversation so we can discuss at the upcoming meeting.

Reply:
Group decided to add a blank line between peer declarations (other than obvious cases where it would harm readability, e.g. where there are 5 methods one after another, all with an implementation of `{}`, just for signatures etc).

@BillWagner: Do you want to make the change consistently in this PR and we'll work out what to do elsewhere?

Reply:
I'll make the changes consistently in the areas touched by this PR.

I'll keep notes and make an open issue for all formatting so we have a record of our preferred formatting. Maybe we can get some community help on that task.